apps/pie-storybook/stories/pie-card-container.stories.ts

```diff
@@ -109,7 +117,8 @@ const Template: TemplateFunction<CardContainerProps> = ({
             target="${target || nothing}"
             rel="${rel || nothing}"
             ?disabled="${disabled}"
-            .aria="${aria}">
+            .aria="${aria}"
+            .isDraggable="${isDraggable}">
```

Comment:
as far as I know, boolean values should be marked with (?) not (.) in lit as the latter is for complex types such as arrays or objects. 

https://github.com/justeattakeaway/pie/discussions/782